robottelo/ui/hosts.py

```diff
@@ -291,3 +291,9 @@ def fetch_host_parameters(self, name, domain_name, parameters_list):
             self.click(tab_locator)
             result[param_name] = self.wait_until_element(param_locator).text
         return result
+
+    def get_host_property(self, parameter_name):
```

Comment:
Why is it redundant and time consuming? You're doing the search and opening of host in the test, my suggestion is to move it inside the helper. And similar to other helpers it can accept not 1 parameter, but a list of parameters, this ways no loops will be needed in tests.